doc/translat.xml

```diff
@@ -0,0 +1,459 @@
+<the semigroup of left translations of <regular transformation 
+ semigroup of size 12, degree 8 with 4 generators>>
+gap> Size(L);
+27
+gap> GeneratorsOfSemigroup(L);
+[ <left translation on <simple transformation semigroup of size 12, 
+     degree 8 with 4 generators>>, 
+  <left translation on <simple transformation semigroup of size 12, 
+     degree 8 with 4 generators>>, 
+  <left translation on <simple transformation semigroup of size 12, 
+     degree 8 with 4 generators>> ]
+]]></Example>
+    </Description>
+  </ManSection>
+<#/GAPDoc>
+
+
+<#GAPDoc Label="XTranslation">
+  <ManSection>
+    <Func Name="LeftTranslation" Arg="L, map"/>
+    <Func Name="RightTranslation" Arg="R, map"/>
+    <Returns>A left or right translation.</Returns>
+    <Description>
+      For the semigroup of left of right translations and a Mapping on the 
+      underlying semigroup, or a transformation acting on the indices of the
+      canonical list of elements of the underlying semigroup,
+      LeftTranslation and RightTranslation return the corresponding translations. 
+      <Example><![CDATA[
+gap> S := RectangularBand(3, 4);;
+gap> L := LeftTranslationsSemigroup(S);;
+gap> s := AsList(S)[1];;
+gap> f := function(x)
+> return s * x;
+> end;;
+gap> map := MappingByFunction(S, S, f);;
+gap> l := LeftTranslation(L, map);
+<left translation on <regular transformation semigroup of size 12, 
+ degree 8 with 4 generators>>
+gap> s ^ l;
+Transformation( [ 1, 2, 1, 1, 5, 5, 5, 5 ] )
+]]></Example>
+    </Description>
+  </ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="TranslationalHull">
+  <ManSection>
+    <Attr Name="TranslationalHull" Arg="S"/>    
+    <Func Name="TranslationalHullSemigroup" Arg="S"/>
+    <Returns>The translational hull of the given semigroup</Returns>
+    <Description>
+      <C>TranslationalHull</C>(<A>S</A>)returns the translational hull of 
+      <A>S</A>. For finite 0-simple semigroups and rectangular bands, explicit 
+      generating sets are known. For small arbitrary semigroups, it may be 
+      possible to calculate the elements of the translational hull. 
+      <C>TranslationalHullSemigroup</C> does the same thing except that it does
+      not attempt to calculate the elements of the translational hull, instead
+      simply creating the object that represents the translational hull.
+      <Example><![CDATA[
+gap> G := Group([(1, 2)(3, 6)(4, 5), (1, 3, 5)(2, 4, 6)]);;
+gap> a := G.1;; b := G.2;;
+gap> mat := [[a, 0, b],
+> [b, a, 0],
+> [0, a, b]];;
+gap> S := ReesZeroMatrixSemigroup(G, mat);;
+gap> H := TranslationalHull(S);
+<translational hull over <0-simple regular semigroup with 6 
+ generators>>
+gap> Size(H);
+67
+]]></Example>
+    </Description>
+  </ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="Bitranslation">
+  <ManSection>
+    <Attr Name="Bitranslation" Arg="H, l, r"/>
+    <Returns>The element of the translational hull corresponding to the given 
+              left and right translations</Returns>
```

Comment:
Just to point out, I could be wrong, but we usually use the `<Returns>` tag to say what kind of GAP object is returned, rather than its mathematical meaning. So this might be more in keeping as:
```suggestion
    <Returns>A translational hull element.</Returns>
```